unit-tests/live/safety/test-safety-metadata.py

```diff
@@ -55,7 +92,37 @@ def check_counter_and_timestamp_increase(frame, fps):
 pipe.stop()
 test.finish()
 
-################# Checking frame counter and timestamp increasing ##################
+################# Checking occupancy metadata required fileds are received ##################
+test.start("Checking Occupancy Stream Metadata received")
+
+cfg = rs.config()
+cfg.enable_stream(rs.stream.occupancy)
+pipe = rs.pipeline()
+pipe.start(cfg)
+iterations = 0
+while iterations < 20:
+    iterations = iterations + 1
+    f = pipe.wait_for_frames()
+    check_occupancy_metadata(f)
+pipe.stop()
+test.finish()
+
+################# Checking point cloud metadata required fileds are received ##################
+test.start("Checking Point Cloud Stream Metadata received")
+
+cfg = rs.config()
+cfg.enable_stream(rs.stream.point_cloud)
+pipe = rs.pipeline()
+pipe.start(cfg)
+iterations = 0
+while iterations < 20:
+    iterations = iterations + 1
+    f = pipe.wait_for_frames()
+    check_point_cloud_metadata(f)
```

Comment:
This check only that the metadata is supported?

Reply:
Yes, and that it can be retrieved